### PR TITLE
Language pack update pt_PT

### PR DESCRIPTION
**Pull request by dfabreu**
This package follows the new orthographic agreement.

### CODE REVIEW

File: data/pt_PT/metadata.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "appWelcomeMessage": "This plugin installs language definitions for the European variant of the Portuguese language in Pentaho",
     "language": "Português Europeu",
     "languageCode": "pt_PT",
-    "pentahoVersion": "6.0.0",
+    "version": "15.07.15",
```
Comment (carlosrusso) [inline data/pt_PT/metadata.json]: The property "version" was removed. Can you please remove this line?

File: data/pt_PT/system/analyzer/resources/messages_pt_PT.properties [hunk 2]
```diff
@@ -1173,69 +1173,12 @@ dlgPageSetupPdfPageFormatA5=A5
...
-actionAddLevel=Adding Level %{0}<TRANSLATE ME>
-actionAddMeasure=Adding Measure %{0}<TRANSLATE ME>
-actionMoveLevel=Moving Level %{0}<TRANSLATE ME>
-actionMoveMeasure=Moving Measure %{0}<TRANSLATE ME>
-actionRemoveLevel=Removing Level %{0}<TRANSLATE ME>
-actionRemoveMeasure=Removing Measure %{0}<TRANSLATE ME>
-popupMenuExportXLSX=To Excel Workbook...<TRANSLATE ME>
-dlgCalcNumClear=Clear<TRANSLATE ME>
-selectSchemaTooltip=Help on selecting data sources<TRANSLATE ME>
-actionUpdateMeasure=Update measure %{0}<TRANSLATE ME>
```
Comment (carlosrusso) [inline data/pt_PT/system/analyzer/resources/messages_pt_PT.properties]: You are not supposed to remove keys. If you do not want to translate them right now, leave them with the <TRANSLATE ME> tag.
Please restore the removed lines.

File: data/pt_PT/system/common-ui/lib/common-ui-5.x_jar/org/pentaho/common/ui/messages/messages_pt_PT.properties [hunk 3]
```diff
@@ -0,0 +1,72 @@
+#
```
Comment (carlosrusso) [inline data/pt_PT/system/common-ui/lib/common-ui-5.x_jar/org/pentaho/common/ui/messages/messages_pt_PT.properties]: The "jar folders"  *-5.x_jar lost the version digit:

common-ui-5.x_jar -> common-ui_jar

You probably made your changes on top on an older master.

**Comment by carlosrusso** [comment]
@nmgomes @dfabreu Please rebase your changes to the current master and solve the conflicts: I cannot merge your changes...

**Comment by nmgomes** [comment]
@carlosrusso Hi Carlos. I made a new branch with the conflicts resolved. Can I make a new pull request with the new branch.

**Comment by carlosrusso** [comment]
@nmgomes No need to ask permission first! Submit the pull request, if for some reason it can't be merged I'll complain.

**Pull request closed without merging**